### ISSUE TITLE
Basic Tabs template does not run on device 

### ISSUE THREAD

**Issue by tattivitorino** [ionic-team/ionic#11932]
**Ionic version:**  (check one with "x")
[ ] **1.x** (For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[X ] **3.x**

**I'm submitting a ...**  (check one with "x")
[X ] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**

Xcode 8 builds the project but do not initiate the app and gives me the error: 

dyld: Symbol not found: ___NSDictionary0__
  Referenced from: /private/var/mobile/Containers/Bundle/Application/D8F1365B-6D9B-4C77-AD04-3A1A67D23F1B/MyApp.app/MyApp
  Expected in: /System/Library/Frameworks/CoreFoundation.framework/CoreFoundation
 in /private/var/mobile/Containers/Bundle/Application/D8F1365B-6D9B-4C77-AD04-3A1A67D23F1B/MyApp.app/MyApp

Running on Device 
IPhone 5s
IOS 8.4
Deployment target on Xcode is 8.0

I also tried to add and remove the CoreFoundation.framework, set it to optional, but it doesn't save the problem.

**Other information:**

It started happening  after i updated both cordova and ionic cli. I created a simple ionic tabs project with nothing but the default. 

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
global packages:

    @ionic/cli-utils : 1.3.0
    Cordova CLI      : 7.0.1 
    Ionic CLI        : 3.3.0

local packages:

    @ionic/app-scripts              : 1.3.7
    @ionic/cli-plugin-cordova       : 1.3.0
    @ionic/cli-plugin-ionic-angular : 1.3.0
    Cordova Platforms               : ios 4.4.0
    Ionic Framework                 : ionic-angular 3.3.0

System:

    Node       : v6.5.0
    OS         : OS X El Capitan
    Xcode      : Xcode 8.1 Build version 8B62 
    ios-deploy : 1.9.1 
    ios-sim    : 5.0.13 
```

any help will be very much appreciated!
tks

**Comment by jgw96** [comment]
Hello! Thanks for opening an issue with us! As this seems like more of a support question I will urge that you ask this question on our [forum](https://forum.ionicframework.com) or on our [slack channel](https://ionicworldwide.herokuapp.com/). Thanks for using Ionic!

**Comment by tattivitorino** [comment]
@jgw96 thanks so much for the response.. and sorry it is in fact a support question! and btw i've already solved the problem! Cordova IOS 4.4.0 does not support IOS 8 anymore. So I downgraded the Cordova platform to IOS 4.3.1.
